.travis.yml

```diff
@@ -2,9 +2,10 @@ language: python
 python:
   - "3.6"
 before_install:
-  - nvm install 4.2
+  - nvm install 6.10.3
```

Comment:
Could you explain the logic behind the bump please and why this version? Note, may be worth adding in a commit message etc

Reply:
This is the version of node that we use in production.

https://github.com/alphagov/digitalmarketplace-docker-base/blob/master/base.docker#L5

Reply:
What Sam said.